### ISSUE TITLE
TabStrip colors

### ISSUE THREAD

**Issue by YKdvd** [telerik/kendo-theme-default#554]
The [tabstrip/_theme.scss file defines some colors at the top of the file](https://github.com/telerik/kendo-theme-default/blob/2241e16303659e4013693828ac6ad8550d3e6ae1/scss/tabstrip/_theme.scss#L1-L6):
```scss
$tabstrip-tabs-color: $accent !default;
$tabstrip-border-color: rgba(0, 0, 0, .08) !default;
$tabstrip-hover-color: blend-multiply($accent, darken($widget-bg, 8%)) !default;
$tabstrip-active-color: $base-text !default;
$tabstrip-active-background-color: desaturate($widget-bg, 1%) !default;
$tabstrip-disabled-color: darken($widget-bg, 27%) !default;
```
However, these never seem to be used anywhere.  Later on, color is set up as in:
```scss
    .k-tabstrip {
        .k-item {
            color: $link-text;
```
"$link-text" is indirectly defined as "$accent" elsewhere, but wouldn't it make sense to have
```scss
color: $tabstrip-tabs-color;
```
directly - that would seem to be the purpose these were defined for?.  That way, anyone overriding can simply do something like:
```scss
$tab-strip-color: [whatever color is desired];
/* other tabs- color variables overriden also */
@import "kendo-theme-default/scss/all"
```
instead of having to override all the tab stylings that define color and keep synced with any changes in the future?

In our case, we want to change the tabs to use the "selected" styling.  It would be helpful if a "$tabstrip-background-color" variable were defined (it could be "inherit"), and explicitly used to define "background-color: $tabstrip-background-color;" in k-item, so that could be overridden as well.

**Comment by IanMuirhead** [comment]
@inikolova 

I am currently trumping these in our custom theme as well - I also set a custom `:hover`, `:focus` and `.k-state-active` 

We also change a number of other attributes, such as: `border-left`, `border-right`, `border-top`, `border-bottom`, `margin-bottom` ...

Our theme ends up skinning the kendo theme to look like the following: 
![tabstrip-example](https://user-images.githubusercontent.com/884395/27393208-85a8094a-565d-11e7-839a-fe0bb352efe0.png)

Related specifically to the color settings on the tabstrip, our use case could be partially covered by the following as well.

`$tabstrip-tab-color`
`$tabstrip-tab-color-hover-focus`
`$tabstrip-tab-color-active`

```
.k-tabstrip > .k-tabstrip-items .k-item {
  color: $tabstrip-tab-color;

  &:hover,
  &:focus {
    color:  $tabstrip-tab-color-hover-focus;
  }
```
```
.k-tabstrip-top>.k-tabstrip-items .k-item.k-state-active {
   color: $tabstrip-tab-color-active
}
```

**Comment by YKdvd** [comment]
I've [forked this repo, and made some changes](https://github.com/YKdvd/kendo-theme-default/commits/8cb55076d0fd20d707225a104901970543a97d99/scss/tabstrip/_theme.scss) to demonstrate what I was thinking of.  I haven't actually tested this yet, and I suspect the usage of $panel-bg and $panel-border should also be variable-ized.  If I get some free time I can try and finish this and make it a pull request.

The hover color actually used ($link-hover-text) is defined as desaturate(darken($accent, 5), 25); I'm not sure if that produces a different color than what was defined (but never used) as $tabstrip-hover-color: blend-multiply($accent, darken($widget-bg, 8%).

**Comment by IanMuirhead** [comment]
That would work for my use case for the hover / active / selected colors.

**Comment by joneff** [comment]
@IanMuirhead @YKdvd 

I've created a pull request (#560) to target the issues outlined here. Few things of note:

* added scoped variables for tabstrip
* item- prefix for the items
* content- for the pane
* the lines for further customization are commented out.

The lines for further customization are the tricky bit: by design the new default theme must have sort of "continuous enclosure" around the selected tab and the related content and the rest of the tabs to look like plain links i.e. by design we haven't provided option for `:hover` border or `:normal` background.

On the variables, prior that change, originally we followed the design assets quite literally, including the `multiply`, `blend` functions. That has proven not to be an easy path, so @yordanov and @lkarakoleva tweaked the designs to achieve similar appearance using tints and shades. Such artifacts may be present in other files.

/cc @gyoshev @inikolova
